src/main/java/com/backbase/ct/bbfuel/configurator/PocketsConfigurator.java

```diff
@@ -96,17 +101,20 @@ public String ingestPocketArrangementForModeOnetoOneAndSetEntitlements(
      * Ingests Pockets for the given user.
      *
      * @param externalUserId External user ID to ingest pockets for.
+     * @return List<Pocket>
      */
-    public void ingestPockets(String externalUserId) {
+    public List<Pocket> ingestPockets(String externalUserId) {
         log.debug("Going to ingest pockets for user [{}]", externalUserId);
-
+        List<Pocket> createdPockets = Collections.synchronizedList(new ArrayList<>());
```

Comment:
`createdPockets` doesn't really need to be `synchronized`.

Reply:
changed